From: Ghislain Antony Vaillant <ghisvail@gmail.com>
Date: Sat, 15 Apr 2017 16:01:30 +0100
Subject: Use packaged MathJax

---
 doc/conf.py | 2 ++
 1 file changed, 2 insertions(+)

Index: python-xarray-0.18.0/doc/conf.py
===================================================================
--- python-xarray-0.18.0.orig/doc/conf.py
+++ python-xarray-0.18.0/doc/conf.py
@@ -170,6 +170,8 @@ napoleon_type_aliases = {
 }
 
 
+mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
+
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates", sphinx_autosummary_accessors.templates_path]
 
